app/presenters/two_factor_authentication/webauthn_selection_presenter.rb

```diff
@@ -5,7 +5,7 @@ def method
     end
 
     def html_class
-      'hide'
+      'display-none'
```

Comment:
This corresponds to above note about 2FA options for signup, and must match the JavaScript logic for showing the option if the browser supports WebAuthn, by removing the class of the same name.

This is also how I discovered the issue described in https://github.com/18F/identity-idp/pull/4638#discussion_r568948967, where a screen-reader-only class could interoperate with `.btn-border`'s own `display: inline-block`, but `display-none` would not.